core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/add-repositories.x86_64.sh

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env bash -eu
 
+CENTOS_EXTRAS=$(cat <<'EOF'
```

Comment:
It looks like we don't need to add this repo.
We have the following command in `download-requirements.sh` to enable the extras repo:

```
# -> rhel-7-server-extras-rpms # for container-selinux package, this repo has different id names on clouds
# About rhel-7-server-extras-rpms: https://access.redhat.com/solutions/3418891

ON_PREM_REPO_ID='rhel-7-server-extras-rpms'
REPO_ID_PATTERN="$ON_PREM_REPO_ID|rhui-rhel-7-server-rhui-extras-rpms|rhui-REGION-rhel-server-extras|rhel-7-server-rhui-extras-rpms" # on-prem|Azure|AWS7.8|AWS7.9
find_rhel_repo_id 'REPO_ID' "$ON_PREM_REPO_ID" "$REPO_ID_PATTERN"
enable_repo "$REPO_ID"
```

Reply:
Without this repo there is an error - https://stackoverflow.com/questions/65878769/cannot-install-docker-in-a-rhel-server

Reply:
```
07:33:33 ERROR cli.engine.ansible.AnsibleCommand - fatal: [ec2-3-249-105-114.eu-west-1.compute.amazonaws.com]: FAILED! => {"changed": false, "changes": {"installed": ["containerd.io", "docker-ce-cli-20.10.8", "docker-ce-rootless-extras-20.10.8", "docker-ce-20.10.8"]}, "msg": "Error: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3(FUSE_3.2)(64bit)\nError: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3(FUSE_3.0)(64bit)\nError: Package: fuse-overlayfs-0.7.2-6.el7_8.x86_64 (epirepo)\n           Requires: libfuse3.so.3()(64bit)\n", "rc": 1, "results": ["Loaded plugins: amazon-id, search-disabled-repos, versionlock\nResolving Dependencies\n--> Running transaction check\n---> Package containerd.io.x86_64 0:1.4.9-3.1.el7 will be installed\n--> Processing Dependency: container-selinux >= 2:2.74 for package: containerd.io-1.4.9-3.1.el7.x86_64\n---> Package docker-ce.x86_64 3:20.10.8-3.el7 will be installed\n---> Package docker-ce-cli.x86_64 1:20.10.8-3.el7 will be installed\n--> Processing Dependency: docker-scan-plugin(x86-64) for package: 1:docker-ce-cli-20.10.8-3.el7.x86_64\n---> Package docker-ce-rootless-extras.x86_64 0:20.10.8-3.el7 will be installed\n--> Processing Dependency: slirp4netns >= 0.4 for package: docker-ce-rootless-extras-20.10.8-3.el7.x86_64\n--> Processing Dependency: fuse-overlayfs >= 0.7 for package: docker-ce-rootless-extras-20.10.8-3.el7.x86_64\n--> Running transaction check\n---> Package container-selinux.noarch 2:2.119.2-1.911c772.el7_8 will be installed\n---> Package docker-scan-plugin.x86_64 0:0.8.0-3.el7 will be installed\n---> Package fuse-overlayfs.x86_64 0:0.7.2-6.el7_8 will be installed\n--> Processing Dependency: libfuse3.so.3(FUSE_3.2)(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n--> Processing Dependency: libfuse3.so.3(FUSE_3.0)(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n--> Processing Dependency: libfuse3.so.3()(64bit) for package: fuse-overlayfs-0.7.2-6.el7_8.x86_64\n---> Package slirp4netns.x86_64 0:0.4.3-4.el7_8 will be installed\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
```

Reply:
I was able to install on RHEL (Azure) without adding any new repos.

Reply:
Also on AWS (image `RHEL-7.9_HVM-20210208-x86_64-0-Hourly2-GP2`)

```
yum-config-manager --enable rhel-7-server-rhui-extras-rpms

DOCKER_CE_REPO_CONF=$(cat <<'EOF'
[docker-ce-stable]
name=Docker CE Stable - centos/7/$basearch/stable
baseurl=https://download.docker.com/linux/centos/7/$basearch/stable
enabled=1
gpgcheck=1
gpgkey=https://download.docker.com/linux/centos/gpg
EOF
)

cat <<< "$DOCKER_CE_REPO_CONF" > /etc/yum.repos.d/docker-ce.repo

yum makecache fast

yum install docker-ce
```

Reply:
@to-bar Same for me. Could you try do that with epiphany? May be it's my local error.

Reply:
I can try next week,